test/list.js

```diff
@@ -82,7 +82,6 @@ describe('list', async function() {
     const schema = reader.metadata.schema;
     assert.equal(schema.length, 7);
     assert.equal(schema[2].name, 'test');
-    assert.equal(schema[2].converted_type, 3);
```

Comment:
I had to remove this test case because it was checking if the type was map or list but I have removed those cases as the types file do not support it

Reply:
I think this test makes a case that we add it back.